demo/src/example/example-page.js

```diff
@@ -8,67 +8,185 @@ import router from '../core/router';
-   * and the corresponding example's content is loaded.
-   *
-   * @param {Element} exampleEl - The button that was clicked.
-   */
-  onExampleClicked(exampleEl) {
-    const parent = exampleEl.parentNode;
-    const category = parent.dataset.category;
-    const exampleIdx = Array.from(parent.children)
-      .filter(child => child.tagName.toLowerCase() === 'button')
-      .indexOf(exampleEl);
-
-    openPlayerModal(Examples[category][exampleIdx]);
+  get keySystems() {
+    const vendor = this.#drmVendorEl.value;
+
+    if (!vendor) {
+      return undefined;
+    }
+
+    const keySystems = {};
+    const certificateUrl = this.#certificateUrlEl.value;
+    const licenseUrl = this.#licenseUrlEl.value;
+
+    if (Drm.vendors.FAIRPLAY === vendor) {
+      keySystems[vendor] = { certificateUrl, licenseUrl };
+    } else {
+      keySystems[vendor] = licenseUrl;
+    }
+
+    return keySystems;
   }
 
-  /**
-   * Handles the submission of a source (URL or URN).
-   *
-   * @param {string} src - The submitted source (URL or URN).
-   */
-  onSourceSubmitted(src) {
+  get loadBarSrc() {
+    return this.#loadBarEl.value;
+  }
+
+  playCustomUrl() {
+    const src = this.#loadBarEl.value;
     const type = src.startsWith('urn:') ? 'srgssr/urn' : undefined;
+    const keySystems = this.keySystems;
+
+    openPlayerModal({ src, type, keySystems });
+  }
 
-    openPlayerModal({ src, type });
+  /**
+   * The HTML template of the load bar.
+   */
+  loadBarTemplate() {
+    return `
+      <div class="fade-in">
+        <div class="load-bar-container">
+          <span class="material-icons-outlined">insert_link</span>
+          <input type="text" id="load-bar" placeholder="Enter a URL or URN to play its content...">
+          <button id="open-settings-btn" title="Open DRM Settings"><span class="material-icons-outlined">settings</span></button>
+        </div>
+        
+        <form id="load-bar-settings" class="hidden" aria-hidden="true">
+          <h3>DRM Settings</h3>
+          <select id="drm-vendor" aria-label="Select a DRM vendor" required>
+            <option value="" disabled selected>Select a DRM vendor</option>
+            <option value="com.widevine.alpha">Widevine</option>
+            <option value="com.apple.fps.1_0">Fairplay</option>
+            <option value="com.microsoft.playready">PlayReady</option>
+          </select>
+          <input type="text" id="license-url-input" placeholder="Enter the license url...">
+          <input type="text" id="certificate-url-input" placeholder="Enter the certificate url...">
+          <button id="clear-settings-btn" class="icon-btn" type="reset"><span class="material-icons-outlined">delete</span>Clear Settings</button>
+          <hr>
+        </form>
+  
+         <button class="icon-btn load-bar-action" id="play-content-btn"><span class="material-icons-outlined">play_circle</span> Play content</button>
```

Comment:
It would be great if the url input is empty the button is not clickable.

Reply:
Good idea, it is now gray out and disabled if there is not a source in the bar.